src/rpc/blockchain.cpp

```diff
@@ -781,7 +781,8 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Calling this again for the same block and a new peer, will cause the response from the previous peer to be ignored.\n"
```

Comment:
4c5505be omit the comma: `s/peer,/peer/`

maybe:
```suggestion
        "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
```